.copier-answers.yml

```diff
@@ -1,7 +1,8 @@
 # Changes here will be overwritten by Copier; NEVER EDIT MANUALLY
-_commit: 2e7a01f
-_src_path: gh:scipp/copier_template
+_commit: bd89f40
+_src_path: ../copier_template/
```

Comment:
Will update again once 
`https://github.com/scipp/copier_template/pull/9` is merged...